.travis.yml

```diff
@@ -26,7 +28,28 @@ gemfile:
   - gemfiles/rails_6_1.gemfile
 matrix:
   exclude:
-    - rvm: 2.4.7
+    - rvm: 2.4
       gemfile: gemfiles/rails_6_0.gemfile
-    - rvm: 2.4.7
+    - rvm: 2.4
       gemfile: gemfiles/rails_6_1.gemfile
+    # Ruby 2.7 uses a `bigdecimal` version that doesn't support BigDecimal.new
+    # that Rails 4.2 uses. See also:
+    # https://github.com/ruby/bigdecimal#which-version-should-you-select
+    - rvm: 2.7
+      gemfile: gemfiles/rails_4_2.gemfile
+    # Ruby 2.7 warning `sqlite3-1.3.13/lib/sqlite3/statement.rb:108: warning: rb_tainted_str_new is deprecated`
+    # and job wil exceed the maximum log length
+    # see also: https://github.com/sparklemotion/sqlite3-ruby/issues/276
```

Comment:
Example: https://travis-ci.org/github/brendon/acts_as_list/jobs/767571757

Reply:
The warning also occurs with the ranked-model.
https://travis-ci.org/github/mixonic/ranked-model/jobs/755121129